styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java

```diff
@@ -487,16 +487,13 @@ void updateShards(final Storage storage,
       final Long usage = entity.getValue();
       LOG.info("Syncing {} -> {}", resource, usage);
       try {
-        int result = (int) (usage / NUM_SHARDS);
```

Comment:
Does this code belong in `StyxScheduler.java`? Can it be moved to `ShardedCounter.java` instead?

Reply:
That’s also weird. Maybe move to Storage?

Reply:
Better address this in #489 if we want to.

Reply:
OK to get it merged for the scope of this PR?